do-win.sh

```diff
@@ -0,0 +1,227 @@
+﻿#!/bin/bash
+
+echo "begin install vim"
+
+choco install vim
+
+# Install plugins
```

Comment:
Everything below this line is the same as `do.sh`. That means we might forget to update one of the versions. Maybe you can create another script with everything below this line and call that script from here and do.sh

Reply:
I work in other option to install over windows without need another script.